crates/bin/pcli/src/command/init.rs

```diff
@@ -106,8 +110,53 @@ impl SoftKmsInitCmd {
     }
 }
 
+#[derive(Debug, clap::Subcommand)]
+pub enum ThresholdInitCmd {
+    /// Use a centralized dealer to create config files for each signer.
+    Deal {
+        /// The minimum number of signers required to make a signature (>= 2).
+        #[clap(short, long)]
+        threshold: u16,
+        /// One file name for each signer, where the dealt config will be saved.
+        #[clap(short, long, value_delimiter = ' ', multiple_values = true)]
+        out: Vec<Utf8PathBuf>,
```

Comment:
The difference would be between running:
```
<command> dir1/config.toml dir2/config.toml ...
```
and
```
<command> dir1/ dir2/ ...
```
I guess it's probably good to avoid having to remember what the name of the config file is, yeah